src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -132,7 +132,8 @@ public DriveTrain() {
       
             // the Field2d class lets us visualize our robot in the simulation GUI.
             fieldSim = new Field2d();
-
+            SmartDashboard.putData("Field", fieldSim);
```

Comment:
This is to get it to display on the robot simulation window (NetworkTables/SmartDashboard/Field)
Not sure if this needs to be continuously ran or just one time though.
See here -> https://docs.wpilib.org/en/latest/docs/yearly-overview/yearly-changelog.html

Reply:
I doubt this is needed, and it is not a good idea to add features to a PR which is not needed and can't be tested. I guess we can leave it for now...

Reply:
Note it does not say it is *needed* to see in the simulation. But again, we can test it later.

Reply:
Looks to me like it is. Should I commit the test program I used?

Reply:
Also it turns out that you do only need to run it one time only.

Reply:
No please leave it as is. PR should be small, quick and as focused as reasonable. This one is just to get the conversion done. Bugs can be fixed later.